### ISSUE TITLE
Bourbon's transition mixin doesn't work with transform

### ISSUE THREAD

**Issue by akheron** [sass/node-sass#1227]
Here's my scss file:
```scss
@import 'bourbon';
body {
    @include transition(transform 0.3s ease);
}
```

With ruby sass I get the following correct output:
```
$ sass test.scss
body {
  -webkit-transition: -webkit-transform 0.3s ease;
  -moz-transition: -moz-transform 0.3s ease;
  transition: transform 0.3s ease; }
```

With node-sass I get the following incorrect output:
```
$ node-sass test.scss
body {
  -webkit-transition: -webkit-transform;
  -moz-transition: -moz-transform;
  transition: transform; }
```

I've debugged this, and it seems to boil down to an issue with variable arguments and iterating over them with `@each`. Here's another test file:

```scss
@mixin test-varargs-and-each($properties...) {
    @each $list in $properties {
        @debug $list;
    }
}
@include test-varargs-and-each(first second third);
```

With ruby sass:
```
$ sass test2.scss
test2.scss:3 DEBUG: first second third
```

With node-sass:
```
$ node-sass test2.scss
test2.scss:3 DEBUG: first

```

So it looks as if the list was somehow truncated to only contain the first item by `@each`.

**UPDATE:** Fixed sass outputs for the bourbon test

**Comment by akheron** [comment]
Version info for the broken case:
``` $ node-sass --version
node-sass       3.4.1   (Wrapper)       [JavaScript]
libsass         3.3.1   (Sass Compiler) [C/C++]
```

It seems to work correctly with node-sass 3.3.3 and libsass 3.2.5.

**Comment by raldred** [comment]
I'm up against this issue too
So the only solution thus far is to downgrade node-sass?

**Comment by akheron** [comment]
I found no other solution than to downgrade node-sass to 3.3.3.

**Comment by brianreavis** [comment]
@raldred @akheron That or override the `transition` mixin: https://github.com/thoughtbot/bourbon/issues/794#issuecomment-152150112

**Comment by xzyfer** [comment]
Thanks for the report. Apologies for the delay I've been away. This is a known issue the latest LibSass https://github.com/sass/libsass/issues/1645

A fix is planned next LibSass release.

**Comment by raldred** [comment]
thanks @brianreavis that works in the interim, thanks.
@xzyfer no problem thanks for the update.

**Comment by xzyfer** [comment]
This is fixed in the upcoming 3.5.0-beta. Subscribe to #1349 for further updates.
